Sources/Redis/Client/RedisClient.swift

```diff
@@ -68,9 +69,32 @@ public struct RedisClientConfig: Codable {
     public var password: String?
 
     /// Create a new `RedisClientConfig`
-    public init(hostname: String = "localhost", port: Int = 6379, password: String? = nil) {
```

Comment:
😢

Reply:
@kjcolley7 I don't want to break anyones project... but this is pre release :( so I am going to have to ask you to adjust as we finalize API sorry. would you be willing to try somthing like 

let hostname = "localhost"
let port = 5679
let connectionString = "redis://\(hostname):\(port)"
let url = URL(string: connectionString)
and using that URL to see if it restores your projects functionality?

Reply:
That works, thanks!

Reply:
thanks for using it!